src/Features/VisualBasic/VBFeaturesResources.resx

```diff
@@ -1051,4 +1051,125 @@ Sub(&lt;parameterList&gt;) &lt;statement&gt;</value>
   <data name="RemoveUnnecessaryImportsDiagnosticTitle" xml:space="preserve">
     <value>Imports statement is unnecessary.</value>
   </data>
+  <data name="TryBlock" xml:space="preserve">
+    <value>Try block</value>
```

Comment:
Should "Try" be localized here. Similar question for other new strings that contain multiple words, including names of  language features or keywords.